<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_GA</class>
 <widget class="QDialog" name="Dialog_GA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一体化RV结构优化</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/DNA.png</normaloff>icons/DNA.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>1 基本参数</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入功率P(w)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_P">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_n_p">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_n">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>行星轮个数n_p</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输入转速n(rpm)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>输出扭矩T_2(N·mm)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_T_2">
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>3 算法参数</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_recopt">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.900000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>种群规模</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_MAXGEN">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>遗传代数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_NIND">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>8000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_GGAP">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>代沟</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>交叉概率</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>变异概率</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_pm">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>4 算法类型</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_q_sorted">
       <property name="text">
        <string>q_sorted</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_awGA">
       <property name="text">
        <string>awGA</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_rwGA">
       <property name="text">
        <string>rwGA</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButton_q_sorted_new">
       <property name="text">
        <string>q_sorted_new</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_NSGA_II">
       <property name="text">
        <string>NSGA-II</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>271</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>2 设计变量范围</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>256</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_z1_max">
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="22" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_r_max">
       <property name="value">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_r_min">
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QSpinBox" name="spinBox_Z_min">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="26" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_L_max">
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_m_max">
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="spinBox_z_g_min">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_z2_max">
       <property name="value">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_B_max"/>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_b_min">
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>转臂轴承节圆直径D_m</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="spinBox_z_g_max">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>短幅系数K_1</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_m_min">
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="1">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="13" column="1">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="21" column="1">
      <spacer name="verticalSpacer_11">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="23" column="1">
      <spacer name="verticalSpacer_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="19" column="1">
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="17" column="1">
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>针齿直径d_z</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>摆线轮齿宽B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>变量</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>针齿中心圆直径D_z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_z1_min">
       <property name="minimum">
        <number>17</number>
       </property>
       <property name="value">
        <number>17</number>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_z_min">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_B_min"/>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_d_z_min">
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_K_1_min">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_D_z_max">
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_d_z_max">
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_b_max">
       <property name="value">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>行星轮齿宽b</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>渐开线齿轮模数m</string>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_L_min">
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="24" column="2">
      <widget class="QSpinBox" name="spinBox_Z_max">
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_K_1_max">
       <property name="value">
        <double>0.800000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_z2_min">
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>圆柱滚子数量Z</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>滚子有效长度L</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>行星轮齿数z_2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>中心轮齿数z_1</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>圆柱滚子直径D_r</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>摆线轮齿数z_g</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <spacer name="verticalSpacer_13">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="spinBox_m_min">
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="spinBox_m_max">
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_design_range">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>初算设计变量范围</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_optimal_calc">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>优化运算</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>661</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>行星轮齿宽</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>针齿中心圆直径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>针齿直径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>摆线轮齿宽</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>短幅系数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>转臂轴承节圆直径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>圆柱滚子直径</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>滚子有效长度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>中心轮齿数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>行星轮齿数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>渐开线齿轮模数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>摆线轮齿数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>圆柱滚子数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传动效率(%)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>体积(mm^3)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>扭转刚度(N·mm/rad)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传动比</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>750</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>传动效率下限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_eta_min"/>
    </item>
    <item>
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>体积上限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_V_max">
      <property name="maximum">
       <double>9999999.990000000223517</double>
      </property>
      <property name="value">
       <double>9999999.990000000223517</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>扭转刚度下限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_K_pie_min">
      <property name="maximum">
       <double>99999999999.990005493164063</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_select">
      <property name="text">
       <string>筛选</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Pareto最优解上限</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_pareto_max">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>9999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>5 Pareto前端</string>
   </property>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>labels/三目标.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>进度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
